.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git/
+.env
+.env.template
+
+.vscode
+Ansible 
```

Comment:
One that _might_ be worth adding is the Dockerfile itself - there's generally little harm in building it into the image (as contrasted with e.g. secret files like .env) but I find that it's advantageous to include it so that changing it doesn't invalidate Docker's cache; if you're ever making a few tweaks to the Dockerfile it can quickly pay for itself!